index.html

```diff
@@ -1,41 +1,249 @@
     <header class="toolbar">
-        <a class="name">Wahid Shadab</a>
-        <nav class="menu">
-            <div class="menu_item"></div>
-            <div class="menu_item"></div>
-            <div class="menu_item"></div>
-        </nav>
+      <a class="name">Wahid Shadab</a>
+      <nav class="menu">
+        <div class="menu_item"></div>
+        <div class="menu_item"></div>
+        <div class="menu_item"></div>
+      </nav>
     </header>
     <section class="headline">
-        <div class="intro">
-            <h3 class="hey">Hey there <br> I'm Shadab</h3>
-            <p class="iam">I'm a Software Developer</p>
-            <p class="about">
-                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-                don’t hestiate to contact me.
-            </p>
-        </div>
-        <ul class="social_media">
-            <li class="fa fa-linkedin fa-2x"><a href="#"></a></li>
-            <li class="fa fa-github fa-2x"><a href="#"></a></li>
-            <li class="fa fa-twitter fa-2x"><a href="#"></a></li>
-            <li class="fa fa-facebook fa-2x"><a href="#"></a></li>
-            <li class="fa fa-youtube fa-2x"><a href="#"></a></li>            
+      <div class="intro">
+        <h3 class="hey">
+          Hey there <br />
+          I'm Shadab
+        </h3>
+        <p class="iam">I'm a Software Developer</p>
+        <p class="about">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+      <ul class="social_media">
+        <li class="fa fa-linkedin fa-2x"><a href="#"></a></li>
+        <li class="fa fa-github fa-2x"><a href="#"></a></li>
+        <li class="fa fa-twitter fa-2x"><a href="#"></a></li>
+        <li class="fa fa-facebook fa-2x"><a href="#"></a></li>
+        <li class="fa fa-youtube fa-2x"><a href="#"></a></li>
+      </ul>
+    </section>
+    <section class="works">
+      <div class="recentworkContainer">
+        <h1 class="recentWork">My Recent Work</h1>
+        <div class="line"></div>
+        <img src="images/recentWork.png" alt="the photo of my recent project" />
+        <h3 class="recentWorkTitle">Multi-Post Stories</h3>
+        <p class="recentWorkDesc">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <ul class="usedTechs">
+          <li class="tech">CSS</li>
+          <li class="tech">HTML</li>
+          <li class="tech">Bootstrap</li>
+          <li class="tech">Ruby</li>
         </ul>
+        <button class="btn" type="button">See Project</button>
```

Comment:
- [ ] It seems that all your buttons (See project) have lost the background color, see the screenshots below 👇 

- Your design
![image](https://user-images.githubusercontent.com/55465133/130124117-c205b3f1-22d0-4b79-8e71-c32b497ed2bd.png)

- Figma design
![image](https://user-images.githubusercontent.com/55465133/130123916-cedc4672-62f6-4d7e-8b37-5f5aa1441ac3.png)

- [ ] I have also tried to resize the window, but still, your cards are getting stuck to the right as you can see on the screenshot above, can you please try to fix these issues 🙏

Reply:
The background is working fine here and there is no problem with the cards. I have messaged you on slack that would be great if you can answer it.